demos/spectre_v1_pht_sa.cc

```diff
@@ -7,6 +7,14 @@
  * SPDX-License-Identifier: BSD-3-Clause OR GPL-2.0
  */
 
+// Causes misprediction of conditional branches that leads to a bounds check
+// being bypassed during speculative execution. Leaks architecturally
+// inaccessible data from the process's address space.
+//
+// PLATFORM NOTES:
+// This program should leak data on pretty much any system where it compiles.
```

Comment:
Is it implicit here that this is "unless you turn on compiler mitigations" (e.g. SLH), or is that worth pointing out? (Same with BTB, I think -- not as sure w.r.t. ret2spec).

If the platform notes only cover the default/expected platform configuration that seems alright with me.

Reply:
Will assume defaults for now, and add/edit if we hit any confusion.